app/src/lib/git/core.ts

```diff
@@ -160,9 +159,8 @@ export async function git(
     combineOutput(process.stdout)
   }
 
-  const result = await withTrampolineEnv(async env => {
-    const sshEnvironment = await getSSHEnvironment()
```

Comment:
Since I added some SSH knowledge to the trampoline, I moved this SSH env variables to `withTrampolineEnv`, cleaning up the code a bit here and in `spawn.ts` 😄